src/clients/Deluge.ts

```diff
@@ -201,6 +224,8 @@ export default class Deluge implements TorrentClient {
 					message: `Injection failed: ${addResult.error.message}`,
 				});
 				return InjectionResult.FAILURE;
+			} else {
+				return InjectionResult.FAILURE;
```

Comment:
want to add some logging here?

Reply:
```suggestion
				return InjectionResult.FAILURE;
				logger.debug({
					label: Label.DELUGE,
					message: `Unknown injection failure: ${newTorrent.name}`,
				});
```

Something like this might work...